Gopkg.toml

```diff
@@ -90,6 +90,10 @@ required = [
   name = "github.com/mholt/archiver"
   version = "=2.1.0"
 
+[[constraint]]
+  name = "github.com/Masterminds/semver"
```

Comment:
We usually use `"github.com/coreos/go-semver/semver"` - not a big deal but maybe keeping it aligned makes sense?

Reply:
But I see that we apparently use this semver package here. Not sure if we want to create a boyscouting issue to align that.

Reply:
> We usually use "github.com/coreos/go-semver/semver" 

Thx good to know! 😄 I didn't realise we used the coreos library.

In this case this package is already vendored because helm uses it. I think we should use the same. Medium term the tiller stuff should go away with Helm 3 or a `kubectl apply` based approach.